Sources/TuistGenerator/GraphViz/GraphVizGenerator.swift

```diff
@@ -1,106 +0,0 @@
-import Foundation
-import GraphViz
-import TSCBasic
-import TuistCore
-import TuistGraph
-import TuistSupport
-
-public protocol GraphVizGenerating {
```

Comment:
I've opted to removing this as after some of the fixes and simplifications, it ended up being nothing more than a proxy to `GraphToGraphVizMapping`